CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## v2.0.0
+
+### Added
+
+- :warning: **[breaking-change]** Add an identifier for each function registration. When using a same identifier, **the new registered function(s) replace(s) the previous one(s)**.
+
+- :warning: **[breaking-change]** Add an identifier for predicates functions to mimic to same behavior as functions registrations. Several predicate functions can now be passed in a single `if()` with a defined identifier. In the same way, if an another `if()` predicate is set with the same identifier, **it will override the previous one**.
+
+- Add an option `deleteFunctionsWhenJobsStarted` allowing to delete the registered functions when the jobs are started.
```

Comment:
What happens if the call is in error? Do functions are removed?

Reply:
Now they are!
ea77ba95f7dc594b766b2e46182850f4ee0ebcc6